http/server.ts

```diff
@@ -693,19 +618,19 @@ export async function serveTls(
  * incoming connections, and handles requests on these connections with the
  * given handler.
  *
- * If the port is omitted from the address, `:80` is used.
+ * If the port is omitted from the ListenOptions, 80 is used.
  *
- * If the host is omitted from the address, the non-routable meta-address
+ * If the host is omitted from the ListenOptions, the non-routable meta-address
  * `0.0.0.0` is used.
  *
  * ```ts
  * import { listenAndServe } from "https://deno.land/std@$STD_VERSION/http/server.ts";
  *
- * const addr = ":4505";
+ * const port = 4505;
  *
  * console.log("server listening on http://localhost:4505");
  *
- * await listenAndServe(addr, (request) => {
+ * await listenAndServe({port}, (request) => {
```

Comment:
```suggestion
 * await listenAndServe({ port }, (request) => {
```

(formatting docs same as `deno fmt` would for non-docs code)